org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/thingTranslator.js

```diff
@@ -86,30 +86,7 @@ function findResource(schema, di, resourceId) {
     return resource;
 }
 
-function getDeviceResource(translator, di, resourceId) {
-    return translator.get(true)
-        .then(response => {
-            return findResource(response, di, resourceId);
-        });
-}
-
-function postDeviceResource(di, resourceId, payload) {
-    if (di === controlId){
-        var putPayload = resourceSchemaToProviderSchema(resourceId, payload);
-
-        return smartThingsHub.putDeviceDetailsAsync(controlId, putPayload)
-            .then((response) => {
-                var schema = providerSchemaToPlatformSchema(response, true);
-
-                return findResource(schema, di, resourceId);
-            });
-    } else {
-        throw new Error('NotFound');
-    }
-}
-
-var controlId;
-var smartThingsHub;
+const switchDeviceDi = "d455d979-d1f9-430a-8a15-61c432eda4a2";
```

Comment:
>430a [](start = 38, length = 4)

why are these hardcoded again in our translators? Typically Iotivity is the one generating these btw -- why is it that we generate these at our layer to begin with?